src/shared/actions/challenge.js

```diff
@@ -85,7 +85,7 @@ function getSubmissionsDone(challengeId, tokenV2) {
  * @return {Promise}
  */
 function registerDone(auth, challengeId) {
-  return getChallengesService(undefined, auth.tokenV2)
+  return getChallengesService(auth.tokenV3, auth.tokenV2)
```

Comment:
Here and below we use created service only to call the updated register / unregister methods, that do not depend on v2 auth token; l believe, we should remove auth.tokenV2 from the calls to `getChallengesService(..)`